magneticod/magneticod/constants.py

```diff
@@ -6,6 +6,7 @@
 ]
 PENDING_INFO_HASHES = 10  # threshold for pending info hashes before being committed to database:
 
-TICK_INTERVAL = 1  # in seconds (soft constraint)
```

Comment:
There's no real reason to have it or to remove it, either way is fine. I probably did it because I don't usually factor constants to one place when I write in Python.